security/sample-template/xebialabs/xlr-pipeline.yaml.tmpl

```diff
@@ -0,0 +1,87 @@
+{{$app := .AppName | kebabcase}}
+{{$sonarServer := "Sonarqube Server"}}
+{{$fortifyServer := "Fortify Server"}}
+{{$checkmarxServer := "Checkmarx Server"}}
+{{$blackduckServer := "Blackduck Server"}}
+apiVersion: xl-release/v1
+kind: Templates
+spec:
+  - name: {{$app}}
+    type: xlrelease.Folder
+    children:
+    - name: {{$app}} security & compliance
+      type: xlrelease.Release
+      description: |
+        This XL Release template shows how to configure security and compliance.
+      tags:
+        - {{$app}}
+        - Security
+        - Compliance
+      phases:
+      - name: Security & Compliance
```

Comment:
really good to see this phase in the overview. I was wondering if we can make it a bit more extensive, so people rally have the feeling that it does something. 
so what if we also add a little hello world application that will be scanned?! Or this almost impossible to do because new projects have to be created in these pipelines?

Some more enrichment with the focus op security (in other phases to make it a bit bigger:
- what if we add an approval task
- what if we add a locked task, and a locked task block (for example a really simple test activity)?
- what if we add a quality gate so it only continues with a certain outcome from the compliance tasks

Reply:
the monolith app can be used if required, it has sonar support already and required Jenkins/Travis jobs and stuff

Reply:
@mayur9991 if you are planning to do this please do as a new improvement story